### DISCUSSION TITLE
Using Postman to query the Web API endpoints

### DISCUSSION THREAD

**Discussion by JasonDeanRoss** [microsoft/electionguard#185] [Q&A]
I've got the Docker container running for the Web API version of the ElectionGuard app, but I'm having issues connecting to it through Postman using the tests included in the repo.  First, I ran the following, as instructed by the [documentation](https://microsoft.github.io/electionguard-api-python/#running-with-docker):

# Pull the image from DockerHub
docker pull electionguard/electionguard-web-api:latest

# Start a container for the API in mediator mode, exposed on port 80 of the host machine
docker run -d -p 80:8000 --env API_MODE=mediator electionguard/electionguard-web-api:latest

Instead of this result:

> After either command, you will find the mediator API running at http://127.0.0.1:8000 and the guardian API at http://127.0.0.1:8001

I found the APIs responding on port 80 and 81 respectively.  And the only endpoint I tried that doesn't return Not Found is the get constants endpoint.  Is there an order of endpoints to run to get the system set up correctly?  I couldn't find any documentation that describes what to do after the Docker container starts up.  Can anyone suggest next steps?

**Comment by lprichar** [comment]
Sorry for the delay in responding to this @JasonDeanRoss .  We've added the following ticket to clarify the readme, do you think that will sufficiently address your concern?  https://github.com/microsoft/electionguard-api-python/issues/206
